src/backend/src/services/ws/index.js

```diff
@@ -98,4 +99,15 @@ export default class WebSocketManager {
     });
     return players;
   }
+
+  findGame(ws) {
+    let gameID = '';
+    // Find a game that hasn't started yet
+    this.games.forEach((value, key) => {
+      if (value.round === 0) {
+        gameID = key;
+      }
+    });
+    ws.send(JSON.stringify({ method: Events.FIND_GAME, gameID }));
```

Comment:
I think it would make sense to start a new game if there aren't any games available to join.